Makefile

```diff
@@ -60,6 +60,6 @@ setup:
 test:
 	$(ISTANBUL) cover node_modules/.bin/_mocha -- --recursive --timeout 10000
 	$(ISTANBUL) check-coverage --branches 100
-ifneq ($(shell node --version | sed 's/[.][^.]*$$//'),v0.10)
+ifeq ($(shell node --version | cut -d . -f 1),v6)
```

Comment:
Great! I often run in a terminal without escape code support, but the string of escape codes would at least draw my attention :)